app/views/result/_enhanced_results_table.html.erb

```diff
@@ -0,0 +1,45 @@
+<%
+  things = %w(
+    document_collections
+    mainstream_browse_pages
+    policies
+    specialist_sectors
+    taxons
+  )
+  enhanced_results_hash = {}
+  things.each do |thing|
+    if enabled?(thing) && side[thing].present?
+      return_array = []
+      side[thing].each do |t|
+        if thing == 'taxons' || thing == 'policies'
+          return_array << [t.humanize.tr("-", " "), '']
+        elsif thing == "mainstream_browse_pages"
+          return_array << [t, link_format("https://gov.uk/browse/#{t}")]
+        else
+          return_array << [t['title'], link_format(t['link'])]
+        end
+        enhanced_results_hash[thing.humanize] = return_array.uniq
+      end
+    end
+  end
```

Comment:
This should not in a view. Does it have tests? what does it even do as it is super unclear? 🍄